src/Symfony/Bundle/FrameworkBundle/Command/CachePoolClearCommand.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\FrameworkBundle\Command;
+
+use Psr\Cache\CacheItemPoolInterface;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use Symfony\Component\HttpKernel\CacheClearer\Psr6CacheClearer;
+
+/**
+ * Clear cache pools.
+ *
+ * @author Nicolas Grekas <p@tchwork.com>
+ */
+class CachePoolClearCommand extends ContainerAwareCommand
+{
+    /**
+     * {@inheritdoc}
+     */
+    protected function configure()
+    {
+        $this
+            ->setName('cache:pool:clear')
+            ->setDefinition(array(
+                new InputArgument('pools', InputArgument::IS_ARRAY, 'A list of cache pools or cache pool clearers'),
+            ))
+            ->setDescription('Clears cache pools')
+            ->setHelp(<<<'EOF'
+The <info>%command.name%</info> command clears the given cache pools or cache pool clearers.
+EOF
+            )
+        ;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        $io = new SymfonyStyle($input, $output);
+        $pools = array();
+        $clearers = array();
+        $container = $this->getContainer();
+        $cacheDir = $container->getParameter('kernel.cache_dir');
+
+        foreach ($input->getArgument('pools') as $id) {
+            $pool = $container->get($id);
+
+            if ($pool instanceof CacheItemPoolInterface) {
+                $pools[$id] = $pool;
+            } elseif ($pool instanceof Psr6CacheClearer) {
+                $clearers[$id] = $pool;
+            } else {
+                throw new \InvalidArgumentException(sprintf('"%s" is not a cache pool nor a cache clearer.', $id));
+            }
+        }
+
+        foreach ($clearers as $id => $clearer) {
+            $io->comment(sprintf('Calling cache clearer: <info>%s</info>', $id));
```

Comment:
I think the clearer/pool specific messages should be displayed only if the output is verbose and maybe a global comment should be added before e.g. "Clearing the cache for pools ...", wdyt?